content/case_histories/mt_isa/survey.rst

```diff
@@ -62,23 +46,12 @@ locations (Powered by: `SimPEG <http://www.simpeg.xyz/>`_).
     :figwidth: 50%
     :name: MIM_DPP_Simulation
 
-    (Top) Pseudo conductivity section (log10) along the geologicalsection.
-    ((Bottom) Recovered 2D conductivity model from the inversion of DCR data.
-    (The true conductivity model is shown in grey scale for reference.
-
-To determine if the survey is well designed we use the criteria in
-{link:general  basics of survey design where we look at the absolute
-difference and relative difference  ?Sarah} The difference sections are shown
-in {figure}. They show xxx If errors are assigned as 5% + floor 0.0001 V then
-the secondary signal from the target is detectable.
-
-Inversion of Synthetic data Following the work {link: general basics of survey
-design} we invert the synthetic data using 2D algorithm.  The parameters used
-in the inversion were:  xxxxx. The Mt. Norvit Horizon is evident.
+    Log10 pseudo-conductivity section along the geological section and the recovered 2D conductivity model from the inversion of DCR data. The top two figures are for the pole-dipole survey and the bottom two images coorrespond to the dipole-pole survey along the same line. In both figures, the true conductivity model is shown in grey scale for reference.
 
-Using 2D forward modelling, we can also simulate current flows that could be
-expected at Mt. Isa. Note in particular the current channeling into the
-conductive Breakaway Shale as well as through the deeper mineralization.
+.. todo::
+  To determine if the survey is well designed we use the criteria for :ref:`detectability <surveyDetectability>` The difference sections are shown in {figure}. They show xxx If errors are assigned as 5% + floor 0.0001 V then the secondary signal from the target is detectable.
 
+  Inversion of Synthetic data Following the work {link: general basics of survey design} we invert the synthetic data using 2D algorithm.  The parameters used in the inversion were:  xxxxx. The Mt. Norvit Horizon is evident.
```

Comment:
can you put the "todo" items in a `.. todo:: ` environment (so we don't forget about them!) - same as the `.. topic::` environment